Mathlib/Algebra/Polynomial/Roots.lean

```diff
@@ -111,6 +111,23 @@ theorem ne_zero_of_mem_roots (h : a ∈ p.roots) : p ≠ 0 :=
 theorem isRoot_of_mem_roots (h : a ∈ p.roots) : IsRoot p a :=
   (mem_roots'.1 h).2
 
+theorem roots_zero_iff_isRoot_bot (hp0 : p ≠ 0) : p.roots = 0 ↔ p.IsRoot = ⊥ := by
+  constructor <;> intro h
+  · ext a
+    simp
```

Comment:
non-terminal simp, please squeeze simp (i.e. put `simp?` and then click the result)